devicetypes/smartthings/zigbee-switch.src/zigbee-switch.groovy

```diff
@@ -35,7 +35,8 @@ metadata {
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0006, 0402, 0B05, FC01, FC02", outClusters: "0003, 0019", manufacturer: "iMagic by GreatStar", model: "1113-S", deviceJoinName: "Iris Smart Plug"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "eWeLink", model: "SA-003-Zigbee", deviceJoinName: "eWeLink SmartPlug (SA-003)"
 		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006", outClusters: "0000", manufacturer: "SONOFF", model: "BASICZBR3", deviceJoinName: "SONOFF Basic (R3 Zigbee)"
-		fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0005, 0006, 0008, 0B05, 1000, FC82", outClusters: "000A, 0019", manufacturer: "innr", model: "SP 222", deviceJoinName: "Innr Smart Plug"
+			//Megaman
+        	fingerprint profileId: "C05E", inClusters: "0000, 0003, 0004, 0005, 0006, FFFF", outClusters: "0019", manufacturer: "MEGAMAN", model: "BSZTM003", deviceJoinName: "INGENIUM® ZB Mains Switching Module"
```

Comment:
Please match the indentation of the existing code. It should be two tabs. Also the extra comment can be removed to follow the pattern used by the other fingerprints.

Reply:
This `deviceJoinName` includes `®` but the other PRs do not. I think you probably want to be consistent and I'd recommend leaving it out.

Reply:
@tpmanley The comment was removed and the name was modified.

Reply:
@tpmanley Add back the existing code.